fieldservice/models/__init__.py

```diff
@@ -1,25 +1,19 @@
-# Copyright (C) 2018 - TODAY, Open Source Integrators
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
 from . import (
     res_company,
     res_config_settings,
     fsm_category,
     fsm_template,
-    fsm_region,
-    fsm_district,
-    fsm_branch,
-    fsm_territory,
+    res_territory,
     fsm_tag,
     fsm_stage,
     fsm_team,
     fsm_person,
     fsm_location,
     fsm_location_person,
     res_partner,
-    res_state,
-    res_country,
     fsm_equipment,
     fsm_order,
-    fsm_route,
```

Comment:
Why is all this code being deleted?

Reply:
@dreispt Same for Here. See the commits https://github.com/OCA/field-service/commit/91a668cfddae75b6127d1c71c7a43cd1e2008d86